CMakeLists.txt

```
@@ -52,16 +52,16 @@ SET(VERSION_PATCH 0-git)
 include(GrVersion) #setup version info
 
 # Minimum dependency versions for central dependencies:
-set(GR_BOOST_MIN_VERSION "1.58")
-set(GR_CMAKE_MIN_VERSION "3.8.0")
-set(GR_MAKO_MIN_VERSION "0.4.2")
-set(GR_PYTHON_MIN_VERSION "3.6.5")
+set(GR_BOOST_MIN_VERSION "1.67")
```

```suggestion
set(GR_BOOST_MIN_VERSION "1.65")
```
I do like Boost bumps, but these two versions don't make such a difference, and we get 18.04 for free.

---

you can `apt install gcc-8` to get GCC 8.4on Ubuntu 18.04, so that shouldn't be a blocker.

about Boost: yeah, 1.65 doesn't really hurt.

Generally: I don't consider Ubuntu 18.04 a viable target; it makes all future maintenance harder, for little added benefit: People who want to use new GNU Radio in 2021 can certainly use the *current* LTS ubuntu.

That being said, of course if we can keep retrocompatibility at little added cost, we could go for it – but there's real overhead in CI, Pybombs, and *forwards* compatibility; my no. 1 reason for wanting Py3.7, for example, is `async`; something we're not using at all currently (and it's unlikely a lot of things using maint-3.9 will magically start doing that), but if an OOT wants to be both GR3.9 and 3.10/master compatible, it cannot use that feature that's really useful to GUI and Web things.

So, I don't agree on the "we really don't need anything in 20.04 that 18.04 doesn't have"; it's just that it was not obvious at this point. You're right, I might be a little paranoid from GR3.7 experience, but then again, the whole hybrid Python dependency in GR3.8 was and is a nightmare, too.  Please don't forget that our maint-* branches do have a longish lifetime after all; if I'm asked to pick whether I want to be able to build something two years longer into the future or with a system from 2 years longer ago, I'm picking the future.